lib/src/tar/export.rs

```diff
@@ -403,18 +407,24 @@ fn impl_export<W: std::io::Write>(
     out: &mut tar::Builder<W>,
     options: ExportOptions,
 ) -> Result<()> {
-    let writer = &mut OstreeTarWriter::new(repo, out, options);
-    writer.write_commit(commit_checksum)?;
+    let writer = &mut OstreeTarWriter::new(out, options);
+    writer.write_commit(repo, commit_checksum)?;
     Ok(())
 }
 
 /// Configuration for tar export.
-#[derive(Debug, Default, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
 pub struct ExportOptions {
     /// Format version; must be 0 or 1.
     pub format_version: u32,
 }
 
+impl Default for ExportOptions {
+    fn default() -> Self {
+        Self { format_version: 1 }
```

Comment:
I am thinking perhaps we should wait a bit before pulling the trigger on this.  The new export format can only be read by rpm-ostree v2022.1, not v2021.14 (I think).

Or well...hmm, OTOH maybe this is a good time to make the change because we'll be doing a new semver bump too.

Hopefully people using this on the client side can quickly update. So...OK!